src/integration-test/java/uk/ac/ebi/tsc/tesk/AuthIgnoreGroupMembershipIT.java

```diff
@@ -0,0 +1,533 @@
+package uk.ac.ebi.tsc.tesk;
+
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import io.kubernetes.client.ApiClient;
+import io.kubernetes.client.util.Config;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Primary;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.ac.ebi.tsc.tesk.TestUtils.getFileContentFromResources;
+
+/**
+ * @author Ania Niewielska <aniewielska@ebi.ac.uk>
+ * <p>
+ * Integration testing of security (authentication and authorisation using OIDC and Elixir groups)
+ * Kubernetes API and OIDC userInfo endpoint are WireMocked
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+@TestPropertySource(locations = {"classpath:application.properties"},
+        properties = {"security.oauth2.resource.user-info-uri = http://localhost:8090",
+                "spring.profiles.active=auth","tesk.api.authorisation.ignoreGroupMembership=true"})
+public class AuthIgnoreGroupMembershipIT {
+
+    @Autowired
+    private MockMvc mvc;
+
+    @Rule
+    public WireMockRule mockElixir = new WireMockRule(8090);
+
+    @Rule
+    public WireMockRule mockKubernetes = new WireMockRule(wireMockConfig().port(9000).usingFilesUnderDirectory("src/integration-test/resources"));
+
+    @TestConfiguration
+    static class KubernetesClientMock {
+        @Bean
+        @Primary
+        public ApiClient kubernetesApiClient() {
+
+            return Config.fromUrl("http://localhost:9000", false);
+
+        }
+    }
+
```

Comment:
I think we can show the point of membership not interfering with ignoreMembership and still not copy all the tests from the previous test suite. Are they really necessary?